src/openforms/forms/api/serializers/form.py

```diff
@@ -248,16 +248,18 @@ def validate(self, attrs):
                 _template = confirmation_email_template
             else:
                 _template = ConfirmationEmailTemplate(**confirmation_email_template)
-            if not _template.is_usable:
+            if _template.is_empty:
                 raise serializers.ValidationError(
                     {
                         "confirmation_email_option": ErrorDetail(
                             _(
-                                "The form specific confirmation email template is not set up correctly and "
-                                "can therefore not be selected."
+                                "The form-specific confirmation email template "
+                                "(subject + content) should not be empty."
                             ),
                             code="invalid",
-                        )
+                        ),
+                        # raise vacuous error b/c the problem concerns email_option + template
+                        "confirmation_email_template": ErrorDetail(""),
```

Comment:
I'm not sure if this is useful, because the user doesn't see it. I think that the updated error is clear. Maybe you can make it even more explicit and say: "In order to use an email template specific to this form, both the subject and the content of the template need to be filled in" ... or something along those lines :P

Reply:
I should have added a more useful comment for this. The `ValidationError` on `confirmation_email_template` creates the error display on the confirmation tab, the `ValidationError` on `confirmation_email_option` creates the error display on the email option dropdown.